### ISSUE TITLE
Crash - 1.16.3 - 1.16-3.0.4-beta - java.lang.NoSuchMethodError: net.minecraftforge.fluids.FluidUtil.getFluidContained

### ISSUE THREAD

**Issue by skruis** [McJty/TheOneProbe#423]
Saw the same error reported but hidden in another post talking about water showing as undefined. Thought I'd create a separate issue. Thank you for all the work you put in! Let me know if you need more info.

Modpack: ATM6 1.2.0
[crash-2020-11-11_06.31.34-client.txt](https://github.com/McJtyMods/TheOneProbe/files/5524007/crash-2020-11-11_06.31.34-client.txt)

Report attached. Here's an excerpt from the report:

```
java.lang.NoSuchMethodError: net.minecraftforge.fluids.FluidUtil.getFluidContained(Lnet/minecraft/item/ItemStack;)Lnet/minecraftforge/common/util/LazyOptional;
	at mcjty.theoneprobe.apiimpl.providers.DefaultProbeInfoProvider.showStandardBlockInfo(DefaultProbeInfoProvider.java:279) ~[?:1.16-3.0.4-beta] {re:classloading}
	at mcjty.theoneprobe.rendering.OverlayRenderer.getWaitingInfo(OverlayRenderer.java:298) ~[?:1.16-3.0.4-beta] {re:classloading}
	at mcjty.theoneprobe.rendering.OverlayRenderer.renderHUDBlock(OverlayRenderer.java:263) ~[?:1.16-3.0.4-beta] {re:classloading}
	at mcjty.theoneprobe.rendering.OverlayRenderer.renderHUD(OverlayRenderer.java:115) ~[?:1.16-3.0.4-beta] {re:classloading}
	at mcjty.theoneprobe.proxy.ClientProxy.renderGameOverlayEvent(ClientProxy.java:110) ~[?:1.16-3.0.4-beta] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_1642_ClientProxy_renderGameOverlayEvent_Pre.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-3.0.3-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:297) ~[eventbus-3.0.3-service.jar:?] {}
	at net.minecraftforge.client.gui.ForgeIngameGui.renderHUDText(ForgeIngameGui.java:634) ~[?:?] {re:classloading}
	at net.minecraftforge.client.gui.ForgeIngameGui.func_238445_a_(ForgeIngameGui.java:187) ~[?:?] {re:classloading}
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:471) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:953) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:randompatches:RandomPatches Minecraft Transformer,pl:mixin:APP:kubejs.mixins.json:MinecraftMixin,pl:mixin:APP:shutupexperimentalsettings.mixins.json:client.MixinMinecraft,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:584) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:randompatches:RandomPatches Minecraft Transformer,pl:mixin:APP:kubejs.mixins.json:MinecraftMixin,pl:mixin:APP:shutupexperimentalsettings.mixins.json:client.MixinMinecraft,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.3-34.1.25.jar:34.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$476/1571125860.call(Unknown Source) [forge-1.16.3-34.1.25.jar:34.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-7.0.1.jar:?] {re:classloading}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-7.0.1.jar:?] {re:classloading}
```

**Comment by McJty** [comment]
That crash makes no sense. That method is clearly present in forge. What version of forge are you using?

**Comment by McJty** [comment]
I checked with people in forge discord. You need to update forge

**Comment by McJty** [comment]
Fixed in 3.0.5
